frontend/src/component/Workspace/ContentItemHeader.jsx

```diff
@@ -20,6 +21,16 @@ export const FileItemHeader = props => {
           </div>
         </div>
       )}
+      {props.showLastEdit && !props.showSearchDetails && (
+        <div className='content__header__modif'>
+          {props.t('Last Modification')}
+        </div>
+      )}
+      {!props.showSearchDetails && (
```

Comment:
I don't see where this props is set, but if it exists, I think the nomenclature is not good ("Search")